lib/rubocop/cop/minitest_cops.rb

```diff
@@ -7,6 +7,7 @@
 require_relative 'minitest/assert_instance_of'
 require_relative 'minitest/assert_respond_to'
 require_relative 'minitest/assert_truthy'
+require_relative 'minitest/assert_empty_literal'
```

Comment:
Please move this `require_relative 'minitest/assert_empty_literal'` after `require_relative 'minitest/assert_empty'` to arrange it in ASCII order.

Reply:
changed